docs/uptime/analytics/open-web-analytics.md

```diff
@@ -0,0 +1,139 @@
+---
+author:
+    name: Douglas Colby
+    email: admin@mulps.pw
+description: ‘How to Install Open Web Analytics on CentOS 6.5 and 7’
+keywords: ‘open, web, analytics,’
+license: '[CC BY-ND 3.0](http://creativecommons.org/licenses/by-nd/3.0/us/)'
+published: 
+modified: 
+modified_by:
+    name: 
+title: 'Open Web Analytics (OWA) on CentOS 6.5 and 7'
+contributor:
+    name: Douglas Colby
+external_resources:
+ - '[OWA Website](http://www.openwebanalytics.com)'
+ - '[OWA Forum](http://www.openwebanalytics.com/?page_id=4)'
+---
+
+##How to Install Open Web Analytics on CentOS 6.5 and 7 (with MySQL back-end)
+
+This guide assume that you have your Linode already set up and running and that you are familiar with the concept of the command line interface (CLI). Since this guide concerns the installation and configuration of software packages all commands are to be run as the root user.
```

Comment:
Any guide on installing `X` should start my explaining what `X` is, and why the reader would want to install it.